Gruntfile.js

```diff
@@ -24,6 +24,16 @@ module.exports = function(grunt) {
           style: 'compressed'
         }
       },
+      tribune: {
```

Comment:
for consistency, let's use `texastribune` for things related to texastribune.org